pkg/apis/core/validation/validation.go

```diff
@@ -2933,8 +2967,8 @@ func validatePullPolicy(policy core.PullPolicy, fldPath *field.Path) field.Error
 
 // validateEphemeralContainers is called by pod spec and template validation to validate the list of ephemeral containers.
 // Note that this is called for pod template even though ephemeral containers aren't allowed in pod templates.
-func validateEphemeralContainers(ephemeralContainers []core.EphemeralContainer, containers, initContainers []core.Container, volumes map[string]core.VolumeSource, fldPath *field.Path, opts PodValidationOptions) field.ErrorList {
-	allErrs := field.ErrorList{}
+func validateEphemeralContainers(ephemeralContainers []core.EphemeralContainer, containers, initContainers []core.Container, volumes map[string]core.VolumeSource, names sets.String, fldPath *field.Path, opts PodValidationOptions) field.ErrorList {
```

Comment:
changin all of these from `podClaimNames` to `names` is bad - I have no idea what it means in most places.  Names of what?

Reply:
Too much sed, not enough brains. I'll bring back the full name.